katcp/test/test_server.py

```diff
@@ -275,8 +285,16 @@ class DeviceTestServerWrong(DeviceTestServer):
         with self.assertRaises(ValueError):
             DeviceTestServerWrong('', 0)
 
+katcp_version = __version__
 
 class test_DeviceServer(unittest.TestCase, TestUtilMixin):
+
+    expected_connect_messages = (
+            r'#version-connect katcp-protocol 5.0-IM',
+            # Will have to be updated for every library version bump
+            r'#version-connect katcp-library katcp-python-%s' % katcp_version,
```

Comment:
I think you are right, it does not apply. The 5.0 bit should stay the same since that is the specified protocol version, although the library default might change at some stage.